### PR TITLE
LPD-3470 Order Manager cannot approve/reject orders with the OOTB Single Approver workflow

### PR DESCRIPTION
**Pull request by andrea-ale-sbarra**
https://liferay.atlassian.net/browse/LPD-3470

### CODE REVIEW

**Comment by andrea-ale-sbarra** [comment]
ci:test:sf

**Comment by andrea-ale-sbarra** [comment]
ci:test:sf

**Comment by andrea-ale-sbarra** [comment]
ci:test:sf

File: modules/apps/portal-workflow/portal-workflow-kaleo-runtime-integration-impl/src/main/java/com/liferay/portal/workflow/kaleo/runtime/integration/internal/WorkflowTaskManagerImpl.java [hunk 1]
```diff
@@ -1107,7 +1107,8 @@ private void _populateAllowedUsers(
 		if ((role.getType() == RoleConstants.TYPE_ACCOUNT) ||
 			(role.getType() == RoleConstants.TYPE_DEPOT) ||
 			(role.getType() == RoleConstants.TYPE_ORGANIZATION) ||
-			(role.getType() == RoleConstants.TYPE_SITE)) {
+			(role.getType() == RoleConstants.TYPE_SITE) ||
+			(role.getType() == RoleConstants.TYPE_ACCOUNT)) {
```
Comment (luca-pellizzon) [inline modules/apps/portal-workflow/portal-workflow-kaleo-runtime-integration-impl/src/main/java/com/liferay/portal/workflow/kaleo/runtime/integration/internal/WorkflowTaskManagerImpl.java:1111]: isn't this the same as line 1107?

**Comment by andrea-ale-sbarra** [comment]
ci:test:sf

**Comment by andrea-ale-sbarra** [comment]
ci:test:commerce-functional

**Review by luca-pellizzon — APPROVED** [review]

**Comment by andrea-ale-sbarra** [comment]
ci:test:relevant

**Comment by andrea-ale-sbarra** [comment]
ci:forward

**Comment by andrea-ale-sbarra** [comment]
FT was removed from this PR and has been sent along with this [PR](https://github.com/liferay-commerce/liferay-portal/pull/4890) because CI uses the virtual instance to run test => the test can only work with the fix of LPD-3419.

**Pull request closed without merging**